### PR TITLE
CSV Sorting

### PR DESCRIPTION
**Pull request by hunterachieng**

### CODE REVIEW

**Review by lakivisi — COMMENTED** [review]
- add a gitignore file and ignore the pyc files
- Can you create it as a cli app using click or docopt

File: main.py [hunk 1]
```diff
@@ -3,6 +3,14 @@
 data = []
 with open('real-estate-sample.csv', 'r') as file:
```
Comment (lakivisi) [inline main.py:4]: this is repetitive you can create a function that takes in file name and mode and updates accordingly

**Review by lakivisi — CHANGES_REQUESTED** [review]

**Comment by hunterachieng** [comment]
I shall update it to a cli app

File: venv/cmd_project/scripts/command_line.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import csv
+from sorting import MergeSort
+def csv_read_details(file_name):
+    data = []
+    with open(file_name, 'r') as file:
+                reader = csv.reader(file)
+                header = next(reader)
+                for row in reader:
+                    data.append(row)
+    sorted = MergeSort(data)
```
Comment (lakivisi) [inline venv/cmd_project/scripts/command_line.py:12]: Separate this out. Have the function just do a simple thing. The read or write functions should not depend on the sort

File: venv/cmd_project/scripts/command_line.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            data_writer = csv.writer(csvfile)
+            data_writer.writerow(header)
+            data_writer.writerows(data)
+        print(f"File created : {file_name}")
+    except IOError as e:
+        print(f"Couldn't write into file at {file_name}. Error: {e}")
+
+# csv_read_details("real-estate-sample.csv", "r")
+
+@click.command()
```
Comment (lakivisi) [inline venv/cmd_project/scripts/command_line.py:29]: Yes this is a correct usage of click

**Review by lakivisi — CHANGES_REQUESTED** [review]
Good progress on the click

**Pull request merged**